hs_err_pid67836.log

```diff
@@ -0,0 +1,1167 @@
+#
```

Comment:
WAT

Reply:
This file is created from the JRE Errors when running locally, didnt notice it! Ive removed the file and added .log files to the gitignore